include/eve/traits/overload/supports.hpp

```diff
@@ -0,0 +1,234 @@
+//======================================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//======================================================================================================================
+#pragma once
+
+#include <eve/logical.hpp>
+#include <eve/detail/raberu.hpp>
+#include <eve/conditional.hpp>
+
+namespace eve
+{
+  //====================================================================================================================
+  //! @addtogroup extensions
+  //! @{
+  //!   @struct options
+  //!   @brief  Wrapper class around bundle of options passed to eve::callable
+  //!
+  //!   eve::callable can be decorated with options. eve::options is used internally to gather all those options
+  //!   in a @raberu settings instance for further processing.
+  //!
+  //!   @tparam Settings @raberu settings type to wrap.
+  //! @}
+  //====================================================================================================================
+  template <rbr::concepts::settings Settings>
+  struct options : Settings
+  {
+    template <rbr::concepts::option... Options>
+    constexpr EVE_FORCEINLINE explicit options(Options && ... opts) : Settings(EVE_FWD(opts) ...) {}
+
+    template <typename... Options>
+    constexpr EVE_FORCEINLINE options(rbr::settings<Options...> const& opts) : Settings(opts) {}
```

Comment:
Any specific reason this is not explicit?

Reply:
I just forgot I guess

Reply:
Still not explicit